src/ConsoleLogger.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Qase\PHPUnit;
+
+class ConsoleLogger
```

Comment:
very good. I suggest improving it.

first, good console logger supports stdout/stderr
second, write/writeln describes low-level operation, what if: `->log`, `->error`?